docs/api.md

```diff
@@ -294,3 +291,80 @@ function mergeProps(stateProps, dispatchProps, ownProps) {
+
+  * [`getDisplayName`] *(Function)*: computes the connector component's displayName property relative to that of the wrapped component. Usually overridden by wrapper functions. Default value: `name => 'ConnectAdvanced('+name+')'`
+
+  * [`methodName`] *(String)*: shown in error messages. Usually overridden by wrapper functions. Default value: `'connectAdvanced'`
+
+  * [`renderCountProp`] *(String)*: if defined, a property named this value will be added to the props passed to the wrapped component. Its value will be the number of times the component has been rendered, which can be useful for tracking down unnecessary re-renders. Default value: `undefined`
+
+  * [`shouldHandleStateChanges`] *(Boolean)*: controls whether the connector component subscribes to redux store state changes. If set to false, it will only re-render on `componentWillReceiveProps`. Default value:  `true`
+
+  * [`storeKey`] *(String)*: the key of props/context to get the store. You probably only need this if you are in the inadvisable position of having multiple stores. Default value: `'store'`
+
+  * [`withRef`] *(Boolean)*: If true, stores a ref to the wrapped component instance and makes it available via `getWrappedInstance()` method. Default value: `false`
+ 
+  * Addionally, any extra options passed via `connectOptions` will be passed through to your `selectorFactory` in the `factoryOptions` argument.
+
+#### Returns
+
+A higher-order React component class that builds props from the store state and passes them to the wrapped component. A higher-order component is a function which accepts a component argument and returns a new component.
+
+##### Static Properties
+
+* `WrappedComponent` *(Component)*: The original component class passed to `connectAdvanced(...)(Component)`.
+
+##### Static Methods
+
+All the original static methods of the component are hoisted.
+
+##### Instance Methods
+
+###### `getWrappedInstance(): ReactComponent`
+
+Returns the wrapped component instance. Only available if you pass `{ withRef: true }` as part of the `options` argument.
+
+#### Remarks
+
+* Since it returns a higher-order component, it needs to be invoked two times. The first time with its arguments as described above, and a second time, with the component: `connectAdvanced(selectorFactory)(MyComponent)`.
+
+* It does not modify the passed React component. It returns a new, connected component, that you should use instead.
+
+#### Examples
+
+##### Inject `todos` of a specific user depending on props, and inject `props.userId` into the action
+```js
+import * as actionCreators from './actionCreators'
+import { bindActionCreators } from 'redux'
+
+function selectorFactory(dispatch) {
+  let state = {}
+  let ownProps = {}
+  let result = {}
+  const actions = bindActionCreators(actionCreators, dispatch)
+  const addTodo = (text) => actions.addTodo(ownProps.userId, text)
+  const memoize = initShallowMemoizer()
```

Comment:
Whoops. It was supposed to be part of the example. Must've got lost. I'll update.